tools/torchfix/torchfix/visitors/deprecated_symbols/__init__.py

```diff
@@ -89,7 +89,7 @@ def __init__(self):
     def leave_ImportFrom(
         self, node: cst.ImportFrom, updated_node: cst.ImportFrom
     ) -> cst.ImportFrom:
-        if node.module.value == "functorch" and all(
+        if node.module is not None and node.module.value == "functorch" and all(
```

Comment:
Nit
```suggestion
        if getattr(node.module, "value", None)  == "functorch" and all(
```

Reply:
`node.module` itself is None, so this will not work.